packages/kubernetes-api/jest.config.ts

```diff
@@ -15,7 +15,7 @@ export default {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|md)$':
       '<rootDir>/src/__mocks__/fileMock.js',
     '\\.(css|less)$': '<rootDir>/src/__mocks__/styleMock.js',
-    'react-markdown': '<rootDir>/../../node_modules/@hawtio/react/node_modules/react-markdown/react-markdown.min.js',
+    'react-markdown': '<rootDir>/../../node_modules/react-markdown/react-markdown.min.js',
     '@patternfly/react-code-editor': path.resolve(__dirname, './src/__mocks__/codeEditorMock.js'),
```

Comment:
Is this directly related to updating @hawtio/react? If not, maybe just push it directly as a chore, rather than including it in this PR?